engine/table/src/main/java/io/deephaven/engine/table/impl/select/AbstractConditionFilter.java

```diff
@@ -168,28 +174,25 @@ public synchronized void init(TableDefinition tableDefinition) {
 
             // check if this is a filter that uses a numba vectorized function
             Optional<QueryScopeParam<?>> paramOptional =
-                    Arrays.stream(params).filter(p -> p.getValue() instanceof NumbaCallableWrapper).findFirst();
+                    Arrays.stream(params).filter(p -> p.getValue() instanceof CallableWrapper).findFirst();
```

Comment:
Does this logic break if I have a vectorizable function in a formula, followed by other params? Should there be only a single param used, or something? Same question applies to `DhFormulaColumn`.

Reply:
It won't, but I think it is a good idea to make the condition explicit.